### DISCUSSION TITLE
Making query to endpoint with GET

### DISCUSSION THREAD

**Discussion by xinspl** [jaredhendrickson13/pfsense-api#355] [Q&A]
Hello, 
In docs all examples are show for array in "data" but for example endpoint **api/v1/system/ca** returns 
`{
    "status": "ok",
    "code": 200,
    "return": 0,
    "message": "Success",
    "data": {
        "ca": [
            {"refid": "asdasdsd",*****},
{"refid": "cfzxcvzxcxz",****}]}}`

how to query that?
i tried simply by **{"refid": "asdasdsd"}** and **{"ca__refid": "asdasdsd"}** always returns empty data tag.
Using postman with GET request and raw JSON in Body. Thaks for help in advance.

**Comment by jaredhendrickson13** [comment]
Hey!

This actually appears to be a bug. The `data` section shouldn't include the `ca` parent key, it should only be the list nested underneath. There's currently not a way to query responses if the `data` field is not a list. I'll open an issue for this and work on a patch.

Thanks!

**Reply by jaredhendrickson13** [reply]
Issue is open at #357.
